### DISCUSSION TITLE
openRID project

### DISCUSSION THREAD

**Discussion by tboye** [opendatateam/udata#2920] [Show and tell]
Hey!

I just wanted to share a project we've recently launched on `udata`: https://data.openrid.co

Our project centralizes regulatory intelligence, offering a user-friendly database for navigating compliance documents. We've just kicked off, but it's an exciting beginning for us. 

Any feedback or suggestions you might have would be gold to us!

**Merci beaucoup for making this awesome platform and fostering a great environment for contributors!** ❤️

**Comment by maudetes** [comment]
Wow nice, seems quite useful! :clap: 
You've curated the catalog manually or could you harvest some existing catalogs?

I see that you've added some additional properties on datasets (in extras?) that you've indexed : topics, type.
You may be interested by the latest work we've made on `Topics` object that have been refactored (see [udata changelog](https://github.com/opendatateam/udata/blob/master/CHANGELOG.md#620-2023-10-26)), and will still be improved.

Do you plan to add community functionalities in the future (comments, "reuse")?

Thank you for sharing the news and congratulations for the launch :rocket:

**Reply by tboye** [reply]
Thank you for your positive feedback and questions.

- Our project revolves around data harvesting and harmonization as in most cases there are no existing catalogs.
- We appreciate community features and we're currently doing some preparations for user registrations as a step towards enabling them.
- The revival of Topics is great news! We're looking to transition to them soon.

Thanks once again!
